README.md

```diff
@@ -0,0 +1,60 @@
+# CoDoc19 :robot:
+
+An open source bot which helps to diagnose symptoms of Covid 19 virus staying at home!
+
+<img src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" data-canonical-src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" width="250" height="500"  />
+
+## Usage
+Currently we have telegram support. So just search CoDoc 19 in telegram and start interacting with it :man_technologist:
+
+### Main Commands
+
+1. To start interacting with the bot :beginner: :
+```
+/start
+```
+After entering this command you will get following three options:
+
+2. To check the likelihood of having COVID 19 :clipboard:
+```
+/test
+```
+You will get several Yes :heavy_check_mark: | No :x: questions. 
+
+3. To check whether you are taking correct measures against COVID 19 :thinking:
+
+```
+/quiz
+```
+
+Several questions with options. You have to choose the right answer to ensure you have been good in taking measures :fist_left:
+
+4. To get current statistics of death, cases, recoveries etc. :chart_with_upwards_trend:
+
+```
+/update
+```
+
+## TODO
+- [ ] Update Command in progress
+- [ ] Support for other networking sites like twitter,facebook etc.
+
+
+## Contributing
+
+1. 🍴 Fork this repo!
+2. **HACK AWAY!** 🔨🔨🔨
+3. 🔃 Create a new pull request.
+
+Feel free to contribute to this project and treat it like your own. 😊
+
+## Contributors
+
+- [theapache64 🚁](https://github.com/theapache64)
+- [pavanjadhaw 🍀](https://github.com/pavanjadhaw)
+
+
+
+## License
+
+[MIT License](https://github.com/sarthology/ovuli/blob/master/license)
```

Comment:
Can u change the license file to https://github.com/teamxenox/codoc19/blob/master/LICENSE . I'll add the file later.